railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt

```diff
@@ -174,6 +174,10 @@
 #
 # Rails.application.config.active_record.commit_transaction_on_non_local_return = true
 
+# Controls when to generate a value for <tt>has_secure_token</tt> declarations.
+#
+# Rails.application.config.active_record.generate_secure_token_on = :initialize
```

Comment:
`config.active_record.generate_secure_token_on = :initialize` should be added to `load_defaults` under `when "7.1"` in `railties/lib/rails/application/configuration.rb`